cypress/integration/seededFlows/adminFlows/badges/badgeAchievements.spec.js

```diff
@@ -5,14 +5,83 @@ describe('Badge Achievements', () => {
 
     cy.get('@user').then((user) => {
       cy.loginAndVisit(user, '/admin/content_manager/badge_achievements');
+
+      cy.findByRole('table').within(() => {
+        cy.findByRole('button', { name: 'Remove' }).click();
+      });
     });
   });
 
-  it('delete a badge achievement', () => {
-    cy.findByText('Remove').as('removeBadgeButton');
+  describe('delete a badge achievement', () => {
+    it('should display confirmation modal', () => {
+      cy.get('.crayons-modal__box > header > p')
+        .contains('Confirm changes')
+        .should('be.visible');
+    });
+
+    it('should display warning text if confirmation text does not match', () => {
+      cy.get('[data-confirmation-modal-target="confirmationTextField"]').type(
```

Comment:
Once we add the aria-label to this input, we could query by  `findByRole` which would help us test the accessibility too 🚀